lib/json-load.js

```diff
@@ -22,12 +22,14 @@ module.exports.readJSON = function(jsonFilename, callback) {
             callback(err);
             return;
         }
+        let json;
         try {
-            const json = jju.parse(data, {mode: 'json'});
-            callback(null, json);
+            json = jju.parse(data, {mode: 'json'});
         } catch (e) {
             callback(`Error in JSON file format: ${jsonFilename} (line ${e.row}, column ${e.column})\n${e.name}: ${e.message}`);
+            return;
```

Comment:
Should we be using `ERR()` here so we get a full stacktrace?

Reply:
Yes! Good call.